app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -14,7 +14,36 @@ def submit_response
     end
 
     def new_offer
-      raise 'Not yet implemented'
+      @application_offer = MakeAnOffer.new(application_choice: @application_choice)
+    end
+
+    def confirm_offer
+      standard_conditions_array = params.dig(:make_an_offer, :standard_conditions)
+      further_conditions_array = [params.dig(:make_an_offer, :first_condition), params.dig(:make_an_offer, :second_condition), params.dig(:make_an_offer, :third_condition), params.dig(:make_an_offer, :further_condition)].reject(&:blank?)
+      complete_conditions_array = [standard_conditions_array, further_conditions_array].compact.reduce([], :|)
```

Comment:
Would `flatten` do the trick here? https://ruby-doc.org/core-2.3.0_preview1/Array.html#method-i-flatten